deps/chakrashim/core/CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+## Contributing Code
```

Comment:
@panuhorsmalahti git submodules don't allow the thirdparty source to be patched, or work with thirdparty deps that are not available via git.